tests/load_obs_data.py

```diff
@@ -0,0 +1,134 @@
+###############################################################################
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+###############################################################################
+
+import sys
+import os
+import csv
+import json
+
+station2geom = {}
+
+
+def station_to_geom(station_id=''):
+    if station2geom == {}:
+        with open('data/metadata/station/stations.geojson') as file:
+            fc = json.loads(file.read())
+            for f in fc['features']:
+                wsid = f['properties']['wigos_id']
+                geom = zip(('X', 'Y', 'Z'), f['geometry']['coordinates'])
+                station2geom[wsid] = {key: val for (key, val) in geom}
+    if station_id:
+        return station2geom[station_id]
+
+
+def walk_path(path):
+    """
+    Walks os directory path collecting all CSV files.
+
+    :param path: required, string. os directory.
+    :return: list. List of csv paths.
+    """
+    file_list = []
+    for root, _, files in os.walk(path, topdown=False):
+        for name in files:
+            if name == 'observations.csv':
+                continue
+            file_list.append(os.path.join(root, name))
+
+    return file_list
+
+
+def _handle_csvs(files, csv_out=[]):
```

Comment:
1. add docstring and function annotation
2. I wonder whether can have this as a locally scoped function within `handle_csv`?

Reply:
I think it can. The logic was pulled from something where I was using the function recursively. Should be all good though